dfw-init.php

```diff
@@ -228,8 +228,11 @@ private function targeting() {
 		}
 
 		// Templates
-		if ( is_single() ) {
+		// change from is-single() to is_singular() to target pages; only define 'inurl' parameter on non-category, and non-home pages
+		if( is_singular() && (!is_post_type_archive() && !is_home() && !is_front_page())) {
 			$targeting['Page'][] = 'single';
+			global $post;
+			$targeting['inURL'][] = $post->post_name;
```

Comment:
Why does this need to be defined to be the post name? Can't inURL targeting be done with the actual URL?

Reply:
> 2. using post-name instead of full URL is more efficient. DFP (or other ad
services) have a character limit on URL targeting (not more than 40 chars),
so for long URLs, or long domains, targeting becomes difficult. Only
including post_name simplifies the process, and reduces errors.

Okay, that makes some sense. However, jquery.dfp.js supplies several targeting options that would appear to make this unnecessary:https://github.com/coop182/jquery.dfp.js#default-url-targeting

Please remove this change from the pull request.